subprojects/docs/src/docs/userguide/migration/upgrading_version_8.adoc

```diff
@@ -49,27 +49,21 @@ This is a potentially breaking change if you are consuming the console output of
 
 === Deprecations
 
-[[custom_configuration_roles]]
-==== Custom configuration roles
-
-// TODO: There is currently no API for users to provide "custom roles" or even
-// any roles at all other than the existing role-mutating methods. We should revisit
-// this deprecation log and section when we introduce a public API for creating
-// locked-role configurations.
-
-Custom roles have been deprecated.
-Use a pre-defined role instead.
-
 [[configurations_allowed_usage]]
 ==== Allowed configurations usage
```

Comment:
```suggestion
==== Mutating core plugin configuration usage
```

We should be specific in the title that this applies to core configurations only
Also there is no "action" in the existing title. Added `Mutating` so we can describe a certain user action which is being deprecated.